package.json

```diff
@@ -84,6 +84,7 @@
     "hmpo-i18n": "5.0.2",
     "hmpo-logger": "6.1.1",
     "jsonwebtoken": "8.5.1",
+    "moment": "^2.29.4",
```

Comment:
Maybe  we can peg the version

Reply:
@idrop I don't think moment needs to be part of this PR, so can we remove it?

(If it's a missing dependency, then let's make that a separate PR and pin the version as @jkunle suggests)